frontend/apps/web/src/pages/all-tests/index.tsx

```diff
@@ -27,7 +27,7 @@ const Header = props => {
         <Text type="heading 5">{t('filecoin.allTests.allTests')}</Text>
 
         <Button
-          onClick={() => navigate('/repository-details')}
+          onClick={() => navigate('/behaviors')}
```

Comment:
I haven't noticed this for the first time, we should
rename this file to /tests

Reply:
Done